ex11/part2/src/timeModule.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+// import {useState} from 'react';
+
+export default function timeModule(prop) {
+
+    return (
+        <input type={"text"} value={prop.text/prop.divBy} onChange={prop.seter} />
```

Comment:
Using the input component here might seem like a good idea for reusability, but it is unnecessary and everything unnecessary in code should go away. Only add extra components/functions etc when it is painful to live without. Not as a scaffolding for the future.